ArduCopter/Parameters.cpp

```diff
@@ -454,6 +454,13 @@ const AP_Param::Info Copter::var_info[] = {
     // @Increment: 0.1
     // @User: Advanced
     GSCALAR(acro_balance_pitch,     "ACRO_BAL_PITCH",   ACRO_BALANCE_PITCH),
+
+    // @Param: SURF_TRACK_MODE
+    // @DisplayName: Surface Tracking Mode
+    // @Description: which surface to track in surface tracking
+    // @Values: 0: Do not track, 1: Ground, 2: Ceiling
```

Comment:
Let's remove the spaces between the ":" and the values.

Reply:
Ok. Will do this.